subsys/bluetooth/controller/ll_sw/ull_llcp_internal.h

```diff
@@ -954,25 +967,39 @@ static inline void pdu_encode_reject_ext_ind(struct pdu_data *pdu, uint8_t rejec
 /*
  * PHY Update Procedure Helper
  */
-void ull_cp_priv_pdu_encode_phy_req(struct pdu_data *pdu);
+void ull_cp_priv_pdu_encode_phy_req(struct pdu_data *pdu, uint8_t rx_phys, uint8_t tx_phys);
```

Comment:
Hmm, yeah, well, I noticed the reject_ext_ind above, which did a similar 'exception to the rule' and since encode_req and encode_rsp need to use data from ctx and conn respectively, I figured it'd be cleaner to do it this way?

Reply:
I updated as you suggested. It looks nicer, thnx